dictionaryutils/__init__.py

```diff
@@ -239,3 +239,19 @@ def resolve_schema(self, obj, root):
             return [self.resolve_schema(item, root) for item in obj]
         else:
             return obj
+
+    def allow_nulls(self):
+        for key, value in self.schema.items():
```

Comment:
since you are not using `key`, you can use this instead: `for node in self.schema.values()`
(also renaming `value` for transparency, see other comment)